src/main/bash/commands/kobman-remove.sh

```diff
@@ -4,17 +4,25 @@ function __kob_remove
 {
     __kobman_echo_red "This operation would remove the KOBman utility and installed environments"
     __kobman_interactive_uninstall || return 1 
-    if [[ -d $KOBMAN_ENV_ROOT ]]; then
-        __kobman_echo_no_colour "Removing dev environment"
-        rm -rf $KOBMAN_ENV_ROOT
-    fi
-
+    # if [[ -d $KOBMAN_ENV_ROOT ]]; then
+    #     __kobman_echo_no_colour "Removing dev environment"
+    #     rm -rf $KOBMAN_ENV_ROOT
+    # fi
+    __kobman_secure_curl "https://raw.githubusercontent.com/$KOBMAN_NAMESPACE/KOBman/master/dist/environments" >> $HOME/env_tmp.txt
```

Comment:
@asa1997  Name the tmp file with what variable or value will be saved in it. Standardized naming for all temp file with a tmp prefix

Reply:
File name has been changed